browser/src/control/Control.Zotero.js

```diff
@@ -27,6 +27,9 @@ L.Control.Zotero = L.Control.extend({
 			});
 		} else {
 			return fetch(url).then(function (response) {
+				if (!response.ok)
+					return that.getCachedOrFetch(url);
```

Comment:
what it does? when request fails it will repeat again and again I assume?
Possibly we want to stop after single retry?